### PR TITLE
first pass at adding more info to launch sweeps

### PR DESCRIPTION
**Pull request by gtarpenning**
## Description

launch sweeps had a CLI update, many new features and a new command to boot. 

## Ticket

https://wandb.atlassian.net/browse/WB-13299

## Checklist

- [X] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
    - says my link is invalid?
- [X] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

File: docs/guides/launch/sweeps-on-launch.md [hunk 1]
```diff
@@ -47,18 +49,18 @@ Programmatically create a W&B Sweep with Launch with the W&B CLI.
 Steps 1 and 3 are the same steps you normally take when you create a W&B Sweep. With the exception that you need to specify the name of the job within your sweep YAML configuration file. 
 :::
 
-For example, in the following code snippet, we specify `wandb/launch_demo/job-source-launch_demo-canonical_job_example.py:v0` for the job value:
+For example, in the following code snippet, we specify `'wandb/jobs/Hello World 2:latest'` for the job value:
```
Comment (bcsherma) [inline docs/guides/launch/sweeps-on-launch.md:63]: Doesn't really have to do with this PR but I:
1. Want spaces in the display name
2. Don't want spaces in any identifiers
I feel like we should have a display name that doesn't change the artifact name.

File: docs/guides/launch/sweeps-on-launch.md [hunk 2]
```diff
@@ -68,18 +70,116 @@ parameters:
     max: 20
     min: 0
     distribution: int_uniform
+
+# Optional/Advanced Scheduler Params:
```
Comment (bcsherma) [inline docs/guides/launch/sweeps-on-launch.md]:
```suggestion
# Optional/advanced scheduler params:
```

File: docs/guides/launch/sweeps-on-launch.md [hunk 3]
```diff
@@ -68,18 +70,116 @@ parameters:
...
+```bash
+wandb launch-sweep <optional config.yaml> --resume_id <sweep id> --queue <queue_name>
+```
+  </TabItem>
+</Tabs>
+
+
+## Create an advanced sweep
+
+Using sweeps with launch enables superior customizability. The sweep scheduling mechanism can be entirely replaced with a job! To do so, create a sweep scheduler job, or point to a public wandb scheduler job. Examples of what is possible with custom sweep scheduler jobs are available in the [wandb/launch-jobs](https://github.com/wandb/launch-jobs) repo under `jobs/sweep_schedulers`. This guide shows how to use the publicly available Wandb Scheduler Job, as well demonstrates a process for creating custom sweep scheduler jobs.
```
Comment (bcsherma) [inline docs/guides/launch/sweeps-on-launch.md]:
```suggestion
Sweeps on Launch are far more customizable than standard sweeps. The sweep scheduling mechanism can be entirely replaced with a job! You can use premade schedulers or implement your own custom scheduler job. Examples of what is possible with custom sweep scheduler jobs are available in the [wandb/launch-jobs](https://github.com/wandb/launch-jobs) repo under `jobs/sweep_schedulers`. This guide shows how to use the publicly available **Wandb Scheduler Job**, as well demonstrates a process for creating custom sweep scheduler jobs. 
```

**Review by bcsherma — APPROVED** [review]
Good stuff big dog. I got genuinely excited about these features reading the docs.

File: docs/guides/launch/sweeps-on-launch.md [hunk 4]
```diff
@@ -8,8 +8,8 @@ import TabItem from '@theme/TabItem';
 
 Launch jobs directly from within W&B locally on your machine or to the compute provider of your choice. Reproduce runs or sweeps directly from the W&B User Interface to launch new experiments and compare results.
```
Comment (ngrayluna) [inline docs/guides/launch/sweeps-on-launch.md:14]: Let's add a quick blurb that describes/distinguishes between the two methods of creating sweeps; 'standard/basic'  vs 'advanced' sweep.

File: docs/guides/launch/sweeps-on-launch.md [hunk 5]
```diff
@@ -68,18 +70,116 @@ parameters:
...
+wandb launch-sweep <path/to/yaml/file> --queue <queue_name> --entity <your_entity>  --project <project_name>
 ```
 
 For more information on W&B Sweeps, see the [Tune Hyperparameters](../sweeps/intro.md) chapter.
 
 
+</TabItem>
+<TabItem value="existing">
+
+  It is also possible to resume a launch-sweep from a previously launched sweep. Although hyperparameters and the training job cannot be changed, scheduler-specific parameters (like `num_workers`) can be, as well as the queue it is pushed to.
```
Comment (ngrayluna) [inline docs/guides/launch/sweeps-on-launch.md]: Let's move 'how to resume a sweep to a different section on this page? The tabs here are meant to show the different ways to create a sweep CLI vs App.

At this point in the doc we don't mention schedule-specific parameters....should we link to that section below...

File: docs/guides/launch/sweeps-on-launch.md (hunk 3 above)
Comment (ngrayluna) [reply docs/guides/launch/sweeps-on-launch.md]: Either in the intro blurb at the top, let's define sweep scheduling mechanism.  for first time readers, it's unclear what it is.

**Comment by ngrayluna** [comment]
@gtarpenning Looking pretty good so far.  Once you get me and @bcsherma's changes in, I'll do some minor word-smithing and merge it in. :)

File: docs/guides/launch/sweeps-on-launch.md (hunk 1 above)
Comment (gtarpenning) [reply docs/guides/launch/sweeps-on-launch.md:63]: I think spaces in the name is how @andrewtruong like it.... I'm just using his jobs.

File: docs/guides/launch/sweeps-on-launch.md (hunk 5 above)
Comment (gtarpenning) [reply docs/guides/launch/sweeps-on-launch.md]: It's sort of another way of creating one, but whatever you think is best!

File: docs/guides/launch/sweeps-on-launch.md (hunk 1 above)
Comment (bcsherma) [reply docs/guides/launch/sweeps-on-launch.md:63]: No I know, it's not really feedback for your PR just felt compelled to document this feeling

File: docs/guides/launch/sweeps-on-launch.md [hunk 6]
```diff
@@ -5,11 +5,22 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 # Sweeps on Launch
```
Comment (ngrayluna) [inline docs/guides/launch/sweeps-on-launch.md:7]: @gtarpenning Can you do me a solid and make sure this is technically accurate?

File: docs/guides/launch/sweeps-on-launch.md [hunk 7]
```diff
@@ -5,11 +5,22 @@ import Tabs from '@theme/Tabs';
...
+By default, the W&B sweep scheduling engine executes runs in a sub-process. The sub-process is responsible for using the optimization algorithm and it determines which combinations of hyperparameters to use.
+
+If you create a custom sweep schedular, however, the schedular you create will instead package the runs and put them on a single launch queue. The custom schedular then polls the runs on the queue and passes that information into its optimization algorithm. The sweep schedular then determines which combinations of hyperparameters to use next.
+
+
+:::tip
+We recommend you create a sweep on launch using the 'basic' method if you are a first time users of sweeps on launch. Use a custom sweeps on launch when the standard W&B scheduling engine does not meet your needs.
+:::
+
+## Create a sweep with a W&B standard schedular
```
Comment (ngrayluna) [inline docs/guides/launch/sweeps-on-launch.md:23]: Thinking behind this section and the next: show how to do the out of the box way here. Then below, show the custom route.

File: docs/guides/launch/sweeps-on-launch.md [hunk 8]
```diff
@@ -68,18 +81,114 @@ parameters:
     max: 20
     min: 0
     distribution: int_uniform
+
```
Comment (ngrayluna) [inline docs/guides/launch/sweeps-on-launch.md:84]: Per comment above, should this chunk of code go to the other section? (Custom)

File: docs/guides/launch/sweeps-on-launch.md [hunk 9]
```diff
@@ -5,11 +5,22 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 # Sweeps on Launch
+Reproduce [runs](../runs/intro.md) or [sweeps](../sweeps/intro.md) directly with W&B Launch.  After you create a queue, W&B Launch creates a sweep schedule job and executes it in the specified environment. When the sweep schedular starts, it uses the hyperparameter configuration you provide and creates multiple sweep runs. You can use the default W&B Sweep scheduling engine or implement your own custom schedular:
 
-Launch jobs directly from within W&B locally on your machine or to the compute provider of your choice. Reproduce runs or sweeps directly from the W&B User Interface to launch new experiments and compare results.
+1. Standard sweep schedular: Use the default W&B Sweep scheduling engine that controls [W&B Sweeps](../sweeps/intro.md). The familiar `bayes`, `grid`, and `random` methods are available.
```
Comment (ngrayluna) [inline docs/guides/launch/sweeps-on-launch.md:12]: Totally happy to change the naming of these two main options...This is just   a suggestion.

**Comment by ngrayluna** [comment]
Noob question, the way I understand this is that folks have two main options: basic/out of the box or a custom schedular. If the latter, they can then either use W&B schedular or a custom schedular?  

If yay, what is the difference between the W&B sched. and a full blown custom sched.?

**Review by ngrayluna — APPROVED** [review]

**Pull request merged**